src/net/http.rs

```diff
@@ -2472,6 +2513,19 @@ impl HttpRequestType {
         }
     }
 
+    /// Formats the query string for a post transaction request.
+    pub fn make_query_string_for_post_tx_endpoint(
+        mempool_admission_check: TransactionAnchorMode,
+    ) -> String {
+        match mempool_admission_check {
+            TransactionAnchorMode::OnChainOnly => "?mempool_admission_check=confirmed".to_string(),
+            TransactionAnchorMode::OffChainOnly => {
+                "?mempool_admission_check=unconfirmed".to_string()
```

Comment:
not tested

Reply:
In line 2074 of `neon_integrations.rs`, I call `submit_tx` with the parameter `mempool_admission_check` set to `TransactionAnchorMode::OffChainOnly`. That function then calls this function. 

Did you run the neon_integrations test with the code coverage tool?